src/components/HomeRigthSidebar.vue

```diff
@@ -10,104 +10,76 @@
-            <BullseyeArrowIco/>
-          </span>
-          <div class="info-box-content">
-          <span class="info-box-text domain-names-target">My target
-          <span class="material-icons float-right vert" id="dropdownMenu2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">more_vert</span>
-   <!--        <div class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenu2">
-            <a href="#" class="color-blue" data-toggle="modal" data-target="#confirmModal">
-            <button class="dropdown-item" type="button">Delete</button>
-            </a>
-          </div>-->
-          </span>
-            <span class="domain-names-list">
-             <small > >reconnes.com, anoth..</small>
+            <span class="domain-names-list text-truncate">
+              <span  class="material-icons float-left mt-1"> chevron_right </span>
+             <small v-for="item2 of item.rootDomains" :key="item2.id">
+              {{item2.root}}, </small>
             </span>
           </div><!-- /.info-box-content -->
         </div><!-- /.info-box -->
       </div><!-- /.targets-list-home -->
-    <div class="modal fade" id="confirmModal">
-        <div class="modal-dialog modal-dialog-centered">
-            <div class="modal-content">
-              <div class="modal-header dialog-without-lines-header">
-                  <!-- PARAMETIZABLE -->
-                  <h5 class="modal-title"><b>
-                       Are you sure you want to delete selected target?</b></h5>
-              </div>
-              <div class="modal-body">
-                  <!-- PARAMETIZABLE -->
-                  <p>Please, confirm the name of the target before delete it</p>
-                  <input autofocus required  style="border-top: none; border-left: none; border-right: none;" class="form-control" placeholder="Target Name">
-              </div>
-              <div class="modal-footer dialog-without-lines-footer">
-                  <button type="button" class="btn btn-primary btn-danger delete_btn" >Delete</button>
-                  <button @click="close()" type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
-              </div>
-            </div><!-- /.modal-content -->
-        </div><!-- /.modal-dialog -->
-    </div><!-- /#confirmation-modal -->
+    <TargetConfirmation></TargetConfirmation>
     </div>
 </template>
 <script>
 import BullseyeArrowIco from '@/components/BullseyeArrowIco.vue'
+import TargetConfirmation from '@/components/Target/TargetConfirmation.vue'
+import { mapState, mapGetters } from 'vuex'
 export default {
   name: 'HomeRigthSidebar',
   components: {
-    BullseyeArrowIco
+    BullseyeArrowIco,
+    TargetConfirmation
+  },
+  computed: {
+    ...mapState('target', ['targetListStore']),
+    ...mapGetters('target', ['getLasTenTargets'])
+  },
+  methods: {
+    orderByNameDesc: function () {
+      function compare (a, b) {

```

Comment:
maybe we can move this to a helper, because we use the same code in different places